projects/wpdx/src/app/rehab-prio/rehab-prio.component.html

```diff
@@ -12,6 +12,15 @@
         <div class='search-bar'>
             <app-adm-search-bar (state)='rpState.navigateToAdm($event)'></app-adm-search-bar>
         </div>
+
```

Comment:
Let's do this in the same search bar - if the input looks like a WPDx Id (use a regular expression), change the list to show "Go To Point <the point id>" instead of the region list.